src/error.rs

```diff
@@ -30,12 +30,16 @@ pub enum Error {
     #[error("Invalid path to custom service account")]
     CustomServiceAccountPath(#[source] std::io::Error),
 
-    /// Failed to parse the application credentials provided
+    /// Failed to parse the application credentials provided in a file
     ///
-    /// By default, the custom service account credentials are parsed from the path pointed to by the
+    /// By default, the custom service account credentials are parsed from the filepath pointed to by the
     /// `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
-    #[error("Application profile provided in `GOOGLE_APPLICATION_CREDENTIALS` was not parsable")]
```

Comment:
How about instead of adding another variant, we just change the error message here to say "Unable to parse custom service account credentials"?

Reply:
I think it's probably okay to remove that. Please make that change!

Reply:
Done 👍